62.CAD/vertex_shader.hlsl

```diff
@@ -169,6 +169,57 @@ bool BezierOBB_PCA(float2 p0, float2 p1, float2 p2, float screenSpaceLineWidth,
     return true;
 }
 
+void BezierOBB_Aligned(float2 p0, float2 p1, float2 p2, float screenSpaceLineWidth, out float2 obbV0, out float2 obbV1, out float2 obbV2, out float2 obbV3)
```

Comment:
move this QuadraticBezier<float_t>instead and rename `screenSpaceLineWidth` to `thickness`

Reply:
done